tensorflow/core/profiler/internal/gpu/device_tracer_rocm.cc

```diff
@@ -339,22 +662,17 @@ class RocmTraceCollectorImpl : public profiler::RocmTraceCollector {
 
     void Export(int32_t device_ordinal, uint64_t start_walltime_ns,
                 uint64_t start_gputime_ns, StepStats* step_stats) {
```

Comment:
Support for StepStats has been recently removed.

Reply:
Correct. It has been removed after the rebase.